tests/test_wellcollection.cpp

```diff
@@ -50,11 +49,11 @@ BOOST_AUTO_TEST_CASE(AddWellsAndGroupToCollection) {
     WellCollection collection;
 
     // Add groups to WellCollection
-    const auto& fieldGroup =  sched.getGroup("FIELD");
+    const auto& fieldGroup =  sched.getGroup2("FIELD", 2);
     collection.addField(fieldGroup, 2, pu);
 
-    collection.addGroup( sched.getGroup( "G1" ), fieldGroup.name(), 2, pu);
-    collection.addGroup( sched.getGroup( "G2" ), fieldGroup.name(), 2, pu);
+    collection.addGroup( sched.getGroup2( "G1", 2 ), fieldGroup.name(), 2, pu);
+    collection.addGroup( sched.getGroup2( "G2", 2 ), fieldGroup.name(), 2, pu);
```

Comment:
I think both the `2` is referred to the report step number. Maybe it is possible to remove one of them in the interface?

The second one should not be needed.

Reply:
That is probably right - thank you!